### PR TITLE
PT-4404 Use helm plugin

### PR DESCRIPTION
**Pull request by aleksandrzhiliaev**

### CODE REVIEW

File: ahoy/rds-exporter/custom-templates/prometheus-servicemonitor.yaml [hunk 1]
```diff
@@ -43,4 +43,4 @@ spec:
     matchLabels:
       app: {{ template "rds-exporter.name" $ }}-{{ $key }}
 {{- end }}
-{{- end }}
+{{- end }}
```
Comment (mauipipe) [inline ahoy/rds-exporter/custom-templates/prometheus-servicemonitor.yaml]: This file should be already present in the template. Without removing the template folder the plugin will not be used

**Comment by spetrashov** [comment]
+1

**Comment by mauipipe** [comment]
+1

**Comment by RTodorov** [comment]
+1

**Pull request merged**